examples/src/main/scala/com/cloudera/livy/apps/WordCountApp.scala

```diff
@@ -0,0 +1,196 @@
+    * @param url The livy server url
+    */
+  def init(url: String): Unit = {
+    scalaClient = new LivyClientBuilder(true).setURI(new URI(url)).build().asScalaClient
+  }
+
+  /**
+    * Uploads the scala-api Jar and the examples Jar from the target directory
+    * @throws FileNotFoundException If either of scala-api Jar or examples Jar is not found
+    */
+  @throws(classOf[FileNotFoundException])
+  def uploadRelevantJarsForJobExecution(): Unit = {
+    val exampleAppJarPath = getPath(this)
+    val scalaApiJarPath =  getPath(scalaClient)
+    uploadJar(exampleAppJarPath)
+    uploadJar(scalaApiJarPath)
+  }
+
+  @throws(classOf[FileNotFoundException])
+  private def getSourceIfNotNull(obj: Object): CodeSource = {
+    val source = obj.getClass.getProtectionDomain.getCodeSource
+    if (source != null) {
+      source
+    } else {
+      throw new FileNotFoundException("Jar Source not found")
+    }
+  }
+
+  @throws(classOf[FileNotFoundException])
+  private def getPath(obj: Object): String = {
+    val source = getSourceIfNotNull(obj)
+    var path = ""
+    if (obj.isInstanceOf[LivyScalaClient]) {
+      path = source.getLocation.getPath
+    } else if (obj == this) {
+      val appTargetFolderPath = new File(source.getLocation.getPath).getParent
+      val targetFolderFiles = new File(appTargetFolderPath).listFiles()
+      if (targetFolderFiles == null || targetFolderFiles.size == 0) {
+        throw new FileNotFoundException("Examples App jar not found in the target folder")
+      }
+      targetFolderFiles.foreach { file =>
+        if (!file.isDirectory && file.getAbsolutePath.contains("jar")) {
+          path = file.getAbsolutePath
+        }
+      }
+    }
+    if (path == "" || !path.contains("jar")) {
+      throw new FileNotFoundException("Jar not found")
+    }
+    path
+  }
+
+  private def uploadJar(path: String) = {
+    val file = new File(path)
+    val uploadJarFuture = scalaClient.uploadJar(file)
+    Await.ready(uploadJarFuture, 40 second) onComplete {
+      case Success(t) => logger.info("Successfully uploaded " + file.getName)
+      case Failure(e) => throw e
+    }
+  }
+
+  /**
+    * Submits a spark streaming job to the livy server
+    * <p/>
+    * The streaming job reads data from the given host and port. The data read
+    * is saved in json format as data frames in the given output path. For simplicity,
+    * the number of streaming batches are 2 with each batch for 20 seconds. The Timeout
+    * of the streaming job is set to 40 seconds
+    * @param host Hostname that Spark Streaming context has to connect for receiving data
+    * @param port Port that Spark Streaming context has to connect for receiving data
+    * @param outputPath Output path to save the processed data read by the Spark Streaming context
+    */
+  def processStreamingWordCount(host: String, port: Int, outputPath: String): ScalaJobHandle[Unit] = {
+    val handle = scalaClient.submit { context =>
```

Comment:
nit: you don't need the `handle` variable if you're just returning it on the next line.